spec/features/rooms/index_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Room, type: :feature do
+  before do
+    @hotel1 = Hotel.create(name: 'Hotel One', location: 'Location One')
+    @hotel2 = Hotel.create(name: 'Hotel Two', location: 'Location Two')
+
+    @room1 = Room.create(suite: 'Suite A', rate: 100, hotel: @hotel1)
+    @room2 = Room.create(suite: 'Suite B', rate: 200, hotel: @hotel2)
+
+    @guest1 = Guest.create(name: 'John Doe', nights: 3)
+    @guest2 = Guest.create(name: 'Jane Doe', nights: 2)
+
+    @stay1 = Stay.create(guest: @guest1, room: @room1)
+    @stay2 = Stay.create(guest: @guest2, room: @room2)
+  end
+
+  describe 'rooms index page' do
+    it 'can see all room data including distinct guest count' do
```

Comment:
This doesn't cover the scope of the user story, which reads: 
```
When I visit the rooms index page
Then I see a list of all rooms
including the room's suite, nightly rate, and the name of the hotel that it belongs to
and the number of guests that have stayed in that room.
```

From this test, you're missing the 2nd room instance and its attributes, including the distinct guest count. (Though, I do see it's in the test marked `show` page. 

I don't say all of this to be a pedant, I'm ok with splitting up and organizing your tests in whatever way suits you best. However, potential employers are going to want to see that you have a sharp attention to detail and can meet their and their clients' expectations with precision. As this and other tests read, I'm not sure I'm seeing that at this point. This is why I am suggesting you copy and paste your user stories in the test file themselves, and go line-by-line in those stories constructing your tests. You can even leave the pasted story in the test for reference.